### DISCUSSION TITLE
ASrock A320 board zero fan control

### DISCUSSION THREAD

**Discussion by desktopstu** [Rem0o/FanControl.Releases#1249] [Q&A]
Hi,  ive used this rather excellent piece of software on two of my pcs (i travel a lot and have PCs here and there) but this one PC i have shows zero fan control and I dont understand why. 
Its an ASrock A320 motherboard, AMD 2700X cpu, MSI 240 AIO cooling, 2 case fans and an AMD RX5600XT with dual fans. 

The fan control software shows GPU fan@750 RPM when idle, Fan 1 @3325 RPM @ idle (i suspect thats the pump) and Fan 2 @933 RPM (could be radiator fans? but thats it. Doesnt allow any control over these and doesnt show the two case fans. All of which are plugged into motherboard headers. 

Any thoughts?

**Comment by Rem0o** [comment]
> but this one PC i have shows zero fan control and I dont understand why.

Different motherboards have different fan control chips. There are a ton of them.  The backend library I'm using doesn't cover 100% of them, as there are so many different ones.
